skimage/measure/_marching_cubes_lewiner_luts.py

```diff
@@ -0,0 +1,529 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012, Almar Klein
```

Comment:
Please remove copyright statements and add contributor names to CONTRIBUTORS.txt